gh-130925: Add `close()` method to `asyncio.StreamReader` (#130929)
python/cpython: Misc/NEWS.d/next/Library/2025-03-06-15-56-12.gh-issue-130925.USr9bm.rst

> :func:`asyncio.StreamReader.close` now exists so that it's possible to
> signal to sub-processes executed via :func:`asyncio.create_subprocess_exec`
> that they may cease generating output and exit cleanly.

Reviewer: ```suggestion
Add :func:`asyncio.StreamReader.close` method. It can signal to sub-processes executed via :func:`asyncio.create_subprocess_exec`
that they may cease generating output and exit cleanly.
```

Author: I think the "so" was a typo? Otherwise I can change this
